Gemfile

```diff
@@ -19,9 +19,9 @@ source "https://rubygems.org"
 
 group :development do
   gem "closure-compiler", "~> 1.1.10"
-  gem "sass", "~> 3.2.19"
+  gem "sass", "~> 3.4.13"
   gem "sprockets-standalone", "~> 1.2.1"
-  gem "sprockets", "~> 2.11.0"
+  gem "sprockets", "~> 3.1.0"
```

Comment:
This can get dropped on crowbar-core, but not from any other component. So we will still have this issue.